lib/Yam/Agama/Pom/GrubEntryEditionPage.pm

```diff
@@ -15,15 +15,13 @@ use testapi;
 sub new {
     my ($class, $args) = @_;
     return bless {
-        number_kernel_line => $args->{number_kernel_line} // 4,
-        max_interval => $args->{number_kernel_line},
+        max_interval => 4,
```

Comment:
I guess we should initialize it to `undef`, this is only needed for ppc64le, but for the rest is not needed to type slowly.
I noticed I introduced a bug in `max_interval => $args->{number_kernel_line}` a time ago which doesn't make sense.

Reply:
updated.